code/modules/cargo/exports/lavaland.dm

```diff
@@ -72,3 +107,34 @@
 	cost = 10000
 	unit_name = "H.E.C.K. helmet"
 	export_types = list(/obj/item/clothing/head/helmet/space/hostile_environment)
+
+//not technically lavaland but this had a useful infrastructure to store them under
+/datum/export/lavaland/gems/diamond
```

Comment:
uuuuuuuuuuhhhhhhhhhhhhhhhh this is the same thing as the thing below it which will probably break shit?

Reply:
fixed